task/backend/scheduler/scheduler_test.go

```diff
@@ -94,7 +94,10 @@ func TestSchedule_Next(t *testing.T) {
 					}
 				}}
 				mockTime := clock.NewMock()
-				mockTime.Set(time.Now())
+				// need to run on a time window which does not include daylight savings
+				// time for testing on systems which do not default to UTC.
+				testTime, _ := time.Parse("2006-01-02", "2020-04-01")
```

Comment:
Could we make both of these values identical? My immediate reaction was to start trying to figure out which was the layout and which was the date

Reply:
Couldn't actually make them identical (2006-01-02 is actually too close the the time change and the test failed) - but I did refactor it to use a `time.RFC3339` as the format string, which should be more clear.